content/release-10-17-0/announcements-10-17-0.md

```diff
@@ -185,41 +185,53 @@ As announced with [release 10.15](/release-10-15-0/announcements-10-15-0) and [r
 
 ### Streaming Analytics
 
+{{< c8y-admon-important >}}
+Cumulocity IoT release 10.17.0 already includes Streaming Analytics release 10.18.0.
+{{< /c8y-admon-important >}}
+
 #### Planned
 
 ##### Deprecation of Machine Learning
 
 As announced with [release 10.16](/release-10-16-0/announcements-10-16-0), the **Machine Learning** block in Analytics Builder is deprecated and will be removed in a future release.
-In addition, the EPL Apps sample "Call another microservice" which uses the Zementis microservice, has been renamed to "Call Zementis microservice".
-This sample is now deprecated and will also be removed in a future release.
 This is in line with the deprecation of [Machine Learning Engine](/release-10-16-0/announcements-10-16-0/#machine-learning-deprecation) announced in release 10.16.
 
-##### Documentation
+#### Implemented
+
+##### New sample for calling another microservice
+<!-- from 10.18 release notes -->
+As announced with [release 10.16](/release-10-16-0/announcements-10-16-0), the EPL Apps sample "Call Zementis microservice" sample has been removed.
+It has been replaced by a new "Call another microservice" sample which now uses the `/health` endpoint of an Apama-ctrl microservice.
+See also [Connecting Apama to other microservices](https://cumulocity.com/guides/10.17.0/streaming-analytics/epl-apps/#microservices) in the *Streaming Analytics guide*.
 
-In a future release, we are going to integrate the Analytics Builder documentation into the Cumulocity IoT documentation.
-Analytics Builder developers will then be able to search and reference the Analytics Builder documentation within the Cumulocity IoT documentation.
+##### Documentation
+<!-- Had to rewrite the "Planned" and move to "Implemented".
```

Comment:
This is actually not an "Important announcement", in the way we usually use it, i.e. for breaking changes or any othe r changes which require customer action. Although they have to change links, bookmarks. The deprecation/removal of the German version could go here but perhaps we move the announcement of the integration of the Analytics Builder guide to the Whats New section?

Reply:
I kept the info on the German version here. And I've shortened the info on the English version. What do you think, @BeateRixen ?

Reply:
Looks good to me. But the sentences on the German version should also go under "Implemented", shouldn´t they?

Reply:
The German sentences are already under "Implemented". Don't let the diff confuse you :-)